includes/commands/chungus.js

```diff
@@ -786,39 +806,52 @@ function stats(msg, args) {
 		getTotalPoints(userId, function(points) {
 			longestChungusHeld(userId, function(longestchung) {
 				secondsAsChungus(userId, function(totalchung) {
-					var embed = new Discord.RichEmbed({
-						thumbnail: {
-							url: chungus_user.avatarURL,
-						},
-						fields: [
-							{
-								name: "**Current Points**",
-								value: `**${points}**`,
-							},
-							{
-								name: "**Total duration**",
-								value: `**${moment
-									.duration(totalchung, "seconds")
-									.humanize()}** (${Math.round(
-									totalchung / 60
-								)} minutes)`,
-								inline: true,
+					currentStreak(userId, function(streakseconds) {
```

Comment:
Id rename streakseconds to currentStreakSeconds, or something similar. Since the embed returns both Longest Streak and Current Streak, it makes sense to differentiate.